morecantile/models.py

```diff
@@ -379,6 +379,7 @@ def zoom_for_res(
         res: float,
         max_z: Optional[int] = None,
         zoom_level_strategy: str = "auto",
+        min_z: Optional[int] = None,
```

Comment:
Added this kwarg last (instead of putting it next to `max_z`) to avoid introducing a breaking change for callers passing  kwargs positionally